packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraCaptureSessionQueueRaceConditionTests.m

```diff
@@ -20,9 +20,15 @@ - (void)testFixForCaptureSessionQueueNullPointerCrashDueToRaceCondition {
       [self expectationWithDescription:@"create's result block must be called"];
   FlutterMethodCall *disposeCall = [FlutterMethodCall methodCallWithMethodName:@"dispose"
                                                                      arguments:nil];
-  FlutterMethodCall *createCall = [FlutterMethodCall
-      methodCallWithMethodName:@"create"
-                     arguments:@{@"resolutionPreset" : @"medium", @"enableAudio" : @(1)}];
+  FlutterMethodCall *createCall =
+      [FlutterMethodCall methodCallWithMethodName:@"create"
+                                        arguments:@{
+                                          @"resolutionPreset" : @"medium",
+                                          @"fps" : @(15),
+                                          @"videoBitrate" : @(200000),
+                                          @"audioBitrate" : @(32000),
```

Comment:
Do these new values matter for the test?

Reply:
no matter. removed.